### DISCUSSION TITLE
Can not commit code review suggestions in specific github repo

### DISCUSSION THREAD

**Discussion by eyalpost** [community/community#8786] [Pull Requests]
I have an issue that consistently appears in a specific github repo. Whenever I try to directly commit a code review suggestion to the PR branch I get the following error:
> This branch has recently been deleted.  Refresh and try again

I click refresh and try again and get the same error message 😢 

![image](https://user-images.githubusercontent.com/2579008/145818826-b84572a2-1326-4f0a-b74d-8a7df0a8c7f9.png)

**Comment by elidrissidev** [comment]
I think the branch the PR was made from got deleted

**Reply by eyalpost** [reply]
No that's not the case. This is consistent in this specific repo for all PRs

**Reply by matkoniecz** [reply]
If problem is with specific repo then you need to link this repo.
